src/game_initialization/multiplayer.cpp

```diff
@@ -682,8 +682,8 @@ void start_local_game_commandline(CVideo& video, const config& game_config, save
 	}
 
 	std::string label = "";
```

Comment:
Also note that this commit appears to effectively render a command-line option superfluous, so if we _are_ accepting the commit, it should also remove all references to the command-line option. (Unless of course the option has other effects elsewhere.)